### DISCUSSION TITLE
2.4.11 Release?

### DISCUSSION THREAD

**Discussion by ChrisBradley** [MahApps/MahApps.Metro#4464] [General]
Hello Mahapps! 

Hope everyone is doing well! I recently started playing with the current update in the develop branch and saw some neat work you done on adding Rounded Corners to MetroWindow via Controlzex... any idea on if there might be a new release anytime soon? We're currently doing something pretty hacky to make this work with MahApps 1.6.4 but it doesn't work great... this seems to be working really nice with whatever is currently in develop branch.

Thanks for everything you folks do! MahApps is still the ONLY framework for making a desktop app look awesome across a decent range of Windows O/S's!
Chris

**Comment by timunie** [comment]
Nightly should have it all in. There is no real due date for the next release as everyting is done in free time and more or less a hobby.

**Comment by punker76** [comment]
https://github.com/MahApps/MahApps.Metro/releases/tag/2.4.11
